addons/medical/functions/fnc_setDead.sqf

```diff
@@ -32,7 +32,7 @@ if (((_reviveVal == 1 && {[_unit] call EFUNC(common,isPlayer)} || _reviveVal ==
     if (_unit getVariable [QGVAR(inReviveState), false]) exitwith {
         if (GVAR(amountOfReviveLives) > 0) then {
             _lifesLeft = _unit getVariable[QGVAR(amountOfReviveLives), GVAR(amountOfReviveLives)];
-            if (_lifesLeft == 0) then {
+            if (_lifesLeft <= 0) exitwith {
```

Comment:
Just anti-error changes, further depuration is needed to find why is happening.

Reply:
Ok I think I can trace what is happening.
Settings:
-preventinstadeath =False
-enablerevive = True
-amountOfReviveLives = 1

Situation:
- _Unit has been revived once with PAK so _LifesLeft = 0
_ _Unit receives accumulation off limb damage.

Process:
1) New damage on lims that is not fatal.
 From: fnc_handleDamage.sqf goes to:
       Line 70   -- fnc_handleDamage_caching.sqf
       Line 109 -- fnc_vitalLoop.sqf
Stays on vital loop and:
   From -- fnc_handleDamage_cahing.sqf goes to
       Line 92 -- fnc_handleDamage_advancedSetDamage.sqf then goes to:
           Lines 28 to 31 -- fnc_setHitPointDamage.sqf then if there is damage accumulation:
                Line 77 -- fnc_setDead.sqf With NO force parameter.
                      Line 31 is OK, Line 32 NOT OK (Unit is not in revive state YET) then
                      Lines 43 to 48 - Unit enters reviveloop and go unconscious -fnc_reviveStateLoop.sqf
                           Line 45 - If PAK is used _ LifesLeft = -1
From there and now on conditions never match with line 35 of fnc_setDead.sqf so:
Unit never dies.

Reply:
Info added.

Reply:
How does this `exitWith` change anything? There is nothing else in this block to execute after it.